webpack.dev.config.ts

```diff
@@ -36,6 +38,29 @@ module.exports = {
   plugins: commonWebpackConfig.plugins,
   module: {
     rules: [{
+      test: /\.svg$/,
```

Comment:
Please use the more specific regex here. (https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/advanced-setup.html#webpack-configuration) `/ckeditor5-[^/]+\/theme\/icons\/[^/]+\.svg$/`

Reply:
Done Thanks!